### ISSUE TITLE
lib/qtrom.ld and unexpected characters?

### ISSUE THREAD

**Issue by biggestsonicfan** [cglmrfreeman/i960-CTOOLS-with-NINDY#3]
At the final link state `I960BASE=~/i960-CTOOLS-with-NINDY/i386-nbsd1-ctools/ ../i386-nbsd1-ctools/bin/lnk960 -o qtrom lib/qtrom.ld kx/kx_init.o basefile.o fp.o float.o fp_flt.o kx/kx.a qt960/qt.a -lckb`, lnk960 is throwing `Unexpected character 13 (decimal) found at lineno...` errors.

I'm not sure what lnk960 doesn't like about the file but it's preventing the final link from happening.

EDIT1: Looking at [ldlex.l](https://github.com/DrItanium/i960/blob/4a6196d815ba73c26f83f8e63e183c80e0e090ba/src/gnu960/src/gld960/i386_nbsd1/ldlex.l#L780) and comparing it to [ldlex.c](https://github.com/DrItanium/i960/blob/4a6196d815ba73c26f83f8e63e183c80e0e090ba/src/gnu960/src/gld960/i386_nbsd1/ldlex.c#L896) they look different but seem to catch what is only a decimal. However, there are no decimals in the lines in qtrom.ld.... this is very perplexing, and perhaps lnk960 needs recompiling?

**Comment by biggestsonicfan** [comment]
So what I ended up doing was stripping the qtrom.ld of all its comments, then renaming it qtrom.c. I then ran qtrom.c through the preprocessor `I960BASE=~/i960-CTOOLS-with-NINDY/i386-nbsd1-ctools/ ../../i386-nbsd1-ctools/bin/ic960 -E qtrom.c > qtrom.ld` and output the file to qtrom.ld. After editing the file to remove the preprocessor comments, it seemed to get further in the compile, so using qtrom_test.ld in commit 3c1886d303f959846b69ddbcd61f5995c0a5c379 will fix this issue.
